### PR TITLE
Support software spotlights migration

### LINKED ISSUE #72: Support hifis.net spotlight migration for RSD
See https://git.gfz-potsdam.de/id2/hifis/rsd/spotlight-migration

### PR DESCRIPTION
**Pull request by Normo**
* Closes #72

### CODE REVIEW

**Comment by Normo** [comment]
Dear @ruester, @tobiashuste, could you please review this PR? Thanks!

**Comment by ruester** [comment]
looks good, thanks :+1:

**Review by ruester — DISMISSED** [review]

**Review by tobiashuste — COMMENTED** [review]
Looks fine. Would be great if the registry link could be updated :)

File: templates/docker-compose.yml.j2 [hunk 1]
```diff
@@ -145,6 +145,19 @@ services:
       - {{ rsd_nginx_dhparam_file_path }}:/etc/ssl/dhparam.pem:ro
     restart: unless-stopped
 
+{% if rsd_migrate_spotlights | default(false) | bool %}
+  initial-spotlights:
+    container_name: initial-spotlights
+    image: {{ rsd_spotlight_migration_image }}
```
Comment (tobiashuste) [inline templates/docker-compose.yml.j2:151]: Also here the `pull` option might be worse to consider like in the other PR.

**Review by tobiashuste — DISMISSED** [review]
Thanks, @Normo. Looks great! :shipit:

**Review by ruester — APPROVED** [review]

**Review by tobiashuste — APPROVED** [review]

**Pull request merged**